app/views/admin_requests/index.html.haml

```diff
@@ -0,0 +1,28 @@
+%h1 Admin Dashboard Page
```

Comment:
I don't think admin_requests folder is required. adminview.html.haml is already in student_requests.